balance/paytable.go

```diff
@@ -0,0 +1,79 @@
+package balance
+
+import (
+	"github.com/LaMonF/FDJ_SLACK/model"
+	"github.com/LaMonF/FDJ_SLACK/utils"
+	"strings"
+)
+
+type Paytable struct{}
+
+// It defines the winning rank which determines the amount of winning money.
+// cf : https://github.com/LaMonF/FDJ_SLACK/issues/10
+type WIN_RANK float64
+
+const (
+	RANK_1 WIN_RANK = 2000000
+	RANK_2 WIN_RANK = 100000
+	RANK_3 WIN_RANK = 1000
+	RANK_4 WIN_RANK = 500
+	RANK_5 WIN_RANK = 50
+	RANK_6 WIN_RANK = 20
+	RANK_7 WIN_RANK = 10
+	RANK_8 WIN_RANK = 5
+	RANK_9 WIN_RANK = 2.20
+	RANK_0 WIN_RANK = 0
+)
+
+func GetwinRanking(result model.LotteryResult, bet model.BetCombo) WIN_RANK {
+	var occurence = utils.ArrayNumberSameOccurence(result.Balls, bet.Balls)
+	if result.LuckyBall == bet.Bonus {
+		if occurence == 0 {
```

Comment:
You should  use a switch here: 

```
switch occurence {
  case 0, 1:
    return RANK_9
  case 2:
    return RANK_7
  ...
}
```

Reply:
Since I do Python, I am not a huge fan of switch cases, I prefer `if` I found it more explicit, but I guess it is a matter of taste...
Any strong opinion @hetrax  and @tigrouxyz  ?